webapp/src/app.tsx

```diff
@@ -2557,7 +2557,7 @@ export class ProjectView
 
     private debouncedSaveProjectName = Util.debounce(() => {
         this.saveProjectNameAsync().done();
-    }, 2000, false);
+    }, 500, false);
```

Comment:
keep 2000 in light mode

Reply:
Is that doable as written? I thought light mode wasn't set till compile was too slow, which wouldn't have occurred when this debounced fnct is created. I guess I can move it to be lazily created in updateHeaderName

I tested on safari / throttled chrome / ios safari simulator / fairly slow android emulator and it seems alright, if someone could create a build so I can test it on chromebook that would be useful